pages/architecture.md

```diff
@@ -18,20 +18,24 @@ together.
 To see if an instrumentation library already exists for your platform, see the
 list of [existing instrumentations]({{ site.github.url}}/pages/existing_instrumentations).
 
-![Architecture overview]({{ site.github.url }}/public/img/architecture-0.png)
-
 Transport
 ---------
 
-Spans sent by the instrumented library must be transported from the services being traced to Zipkin collectors.
-There are two primary transports: Scribe and Kafka. Scribe is shown in the figure. See [Span Receivers]({{ site.github.url }}/pages/span_receivers) for more information.
+Tracing data sent by the instrumented library must be transported from the services being traced to Zipkin collectors.
+There are two primary transports: HTTP and Kafka. Scribe can also be used.
+
+![Architecture overview]({{ site.github.url }}/graphs/architecture/graph.png)
+There are five components that make up Zipkin:
```

Comment:
this will scare people as there's already a stereotype zipkin is hard to install. Maybe change this sentence to..

While zipkin can run in a single process, there are N components that make up Zipkin architecture